packages/nimble-components/src/number-field/index.ts

```diff
@@ -74,6 +74,7 @@ const nimbleNumberField = NumberField.compose<NumberFieldOptions>({
             appearance="ghost"
             content-hidden
             tabindex="-1"
+            aria-hidden="true"
```

Comment:
I am surprised we don't already have tests to validate these attributes. Would you mind at least adding coverage for `tabindex` and `aria-hidden` to assert the intended behavior that you describe in the PR description?